_maps/map_files/cyberiad/z2.dmm

```diff
@@ -9589,8 +9589,8 @@
 /obj/structure/rack,
 /obj/item/ammo_box/magazine/m12g,
 /obj/item/ammo_box/magazine/m12g,
-/obj/item/ammo_box/magazine/m12g/stun,
-/obj/item/ammo_box/magazine/m12g/stun,
```

Comment:
I don't think these should be changed since it's on Z level 2

Reply:
Ah, well I think it was done because they were removed entirely, so that would throw issues. Probably a decent idea to keep around. If a Head or Maint wants these, I'll fix this.